packages/browser/README.md

```diff
@@ -0,0 +1,170 @@
+# Backtrace Browser
+Backtrace error reporting tool for browser side JavaScript
+ 
+## Table of Contents
+1. [Integration Example](#integration-example)
+1. [Features](#features)
+    - [File Attachments](#file-attachments)
+    - [Metrics](#metrics)
+    - [Breadcrumbs](#breadcrumbs)
+    - [Callback Methods](#callback-methods)
+1. [Documentation](#documentation)
+
+## Integration Example
+```js
+// Import the BacktraceClient from @backtrace/browser with your favoriate package manager.
+import { BacktraceClient } from "@backtrace/browser";
+
+// BacktraceClientOptions
+const options = {
+    // Name of the website/application
+    name: "MyWebPage",
+    // Version of the website
+    version: "1.2.3",  
+    // Submission url
+    url: "https://submit.backtrace.io/<universe>/<token>/json",
```

Comment:
Where can we get universe, token?
```
// Universe is the subdomain of Backtrace hosted instance (universe.backtrace.io)
// Token can be found in Project Settings/Submission tokens
```

Reply:
Can we add these two comments to the code snippet, to direct the user to project settings?